antiope-athena/src/Antiope/Athena.hs

```diff
@@ -0,0 +1,78 @@
+module Antiope.Athena
+  ( query
+  )where
+
+import           Control.Lens
+import           Control.Monad
+import           Control.Monad.Catch          (catch)
+import           Control.Monad.Morph          (hoist)
+import           Control.Monad.Trans.AWS      hiding (await, send)
+import           Control.Monad.Trans.Resource
+import qualified Data.ByteString              as BS
+import           Data.ByteString.Lazy         (ByteString, empty)
+import           Data.Conduit                 hiding (await)
+import           Data.Conduit.Binary          (sinkLbs)
+import           Data.Monoid                  ((<>))
+import           Data.Text                    (Text)
+import           Network.AWS                  (Error (..), MonadAWS,
+                                               ServiceError (..), await, send)
+import           Network.AWS.Athena
+import           Network.AWS.Athena.Types
+import           Network.AWS.Athena.Waiters
+import           Network.AWS.Data
+import           Network.AWS.Data.Body        (_streamBody)
+import           Network.AWS.Waiter
+import           Network.HTTP.Types.Status    (Status (..))
+
+
+query :: (MonadResource m, MonadAWS m)
+            => Text
+            -> Text
+            -> Text
+            -> Text
+            -> m [Row]
+query location database qstring clientRequestToken = do
+  let config = resultConfiguration location
```

Comment:
Maybe this should come in as `ResultConfiguration` (https://hackage.haskell.org/package/amazonka-athena-1.6.0/docs/Network-AWS-Athena-Types.html#t:ResultConfiguration)

And `database` can be `QueryExecutionContext`? (https://hackage.haskell.org/package/amazonka-athena-1.6.0/docs/Network-AWS-Athena-Types.html#t:QueryExecutionContext)

This would help the `Text -> Text -> Text -> Text` situation.